example/example-graph/src/main/java/uk/gov/gchq/gaffer/example/operation/GetElementsBySeedExample.java

```diff
@@ -24,15 +24,15 @@
 import uk.gov.gchq.gaffer.operation.data.EdgeSeed;
 import uk.gov.gchq.gaffer.operation.data.ElementSeed;
 import uk.gov.gchq.gaffer.operation.data.EntitySeed;
-import uk.gov.gchq.gaffer.operation.impl.get.GetElementsBySeed;
+import uk.gov.gchq.gaffer.operation.impl.get.GetElements;
 
 public class GetElementsBySeedExample extends OperationExample {
```

Comment:
Perhaps we could merge GetElementsBySeedExample and GetRelatedElementsExample into GetElementsExample. Similarly with Entities and Edges.